src/robusta/core/sinks/opsgenie/opsgenie_sink.py

```diff
@@ -96,6 +94,23 @@ def __to_description(self, finding: Finding, platform_enabled: bool) -> str:
 
         return f"{description}{self.__enrichments_as_text(finding.enrichments)}"
 
+    def __to_details(self, finding: Finding, platform_enabled: bool) -> dict:
```

Comment:
I think the `platform_enabled` parameter is not used

Reply:
Indeed, removed